demos/colab/demo.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import glob
+import os
+import tempfile
+
+import numpy as np
+from distance_function import euclidean_distance, iou
```

Comment:
this module is available under `norfair.distances` no need to repeat it in the demo

Reply:
but be careful with the distance threshold for iou, the implementation provided by norfair is bound to `[0,1]` where 1 means that the boxes don't touch and 0 means the boxes are the same

Reply:
I changed these distance functions to create_normalized_mean_euclidean_distance like you suggest me on the HF demo.